__tests__/app.test.js

```diff
@@ -33,11 +33,11 @@ describe("GET /api/topics", () => {
       .then(({ body }) => {
         const { topics } = body;
         topics.forEach((topic) => {
```

Comment:
I'm hoping you've had feedback on this already (if it was me and I missed it, sorry). When using forEach you should assert a length of the array as this will try to iterate over an empty array.